src/profile/components/emailEditor/__tests__/EmailEditor.test.tsx

```diff
@@ -27,6 +27,19 @@ import { EditDataType, EmailValue } from '../../../helpers/editData';
 import i18n from '../../../../common/test/testi18nInit';
 import RenderChildrenWhenDataIsComplete from '../../../../common/test/RenderChildrenWhenDataIsComplete';
 import getEmailsFromNode from '../../../helpers/getEmailsFromNode';
+import { mockProfileCreator } from '../../../../common/test/userMocking';
+import { AMRStatic } from '../../../../auth/useProfile';
+
+type TestAmr = Extract<AMRStatic, 'google'> | 'foobar';
```

Comment:
Instead of "foobar", could the test amr be something more descriptive, like perhaps "tunnistus-suomi-fi"?

Reply:
This is intentionally silly value so it is not mistaken to the actual value.

Reply:
It could still have some meaning. Perhaps something like "fake-suomi-fi-amr-for-test"...

Reply:
Switched to a value defined in "AmrStatic", because it is most convenient. Switching to a value longer than 10 chars will trigger linting error "sonarjs/no-duplicate-string"